packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/BackButton/BackButton.js

```diff
@@ -11,79 +10,24 @@ export const StyledBackButtonWrapper = styled.div`
   position: relative;
   flex: 0 0 33%;
 `
-
-// Firefox returns CSS.supports('width', 'max-content') as false
-// even though CanIUse reports it is supported by Firefox
-// Only the vendor prefixed -moz-max-content returns true
-export const StyledBackButtonToolTip = styled.span`
-  bottom: '-100%';
-  box-sizing: border-box;
-  ${props => props.theme.dark ? 
-    css`color: ${props.theme.global.colors['accent-1']};` :
-    css`color: ${props.theme.global.colors.brand};`
-  }
-  font-size: 0.9em;
-  left: 0;
-  padding: 1em 0;
-  position: absolute;
-  width: max-content;
-  width: -moz-max-content;
-
-  .rtl & {
-    left: auto;
-    right: 0;
-  }
-  `
 function BackButton({
-  autoFocus = false,
   canUndo = false,
-  disabled = false,
   onClick = () => true,
-  persistAnnotations = false,
   theme = {
     dark: false
   }
```

Comment:
Not required for this PR, but I think you can replace this prop with a `useTheme` hook in the body of the component.

Reply:
Good point! I'll update before merging